src/gui/SampleLoader.cpp

```diff
@@ -101,6 +115,8 @@ std::unique_ptr<SampleBuffer> SampleLoader::createBufferFromFile(const QString&
 
 std::unique_ptr<SampleBuffer> SampleLoader::createBufferFromBase64(const QString& base64, int sampleRate)
 {
+	if (base64.isEmpty()) { return std::unique_ptr<SampleBuffer>(); }
```

Comment:
```suggestion
	if (base64.isEmpty()) { return nullptr; }
```
Though, if the general idea is "return nullptr if input is invalid", maybe you'd want it to be this instead?
```cpp
if (base64.isEmpty() || sampleRate == 0) { return nullptr; }
```